portfolio/src/main/java/com/google/sps/servlets/Chat.java

```diff
@@ -11,30 +11,30 @@
 @WebServlet("/chat")
 public class Chat extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    String jsonQuestions = convertToJsonUsingGson(randomQuestion());
-    response.getWriter().println(jsonQuestions);
-  }
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
```

Comment:
it looks like you changed the formatting to 4 spaces, can you change the formatting back to 2 spaces indentation for all files?